Samples/Samples/Helpers/ViewHelpers.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Xamarin.Forms;
+
+namespace Samples.Helpers
+{
+    public static class ViewHelpers
+    {
+        public static Rectangle GetAbsoluteBounds(this Xamarin.Forms.View element)
+        {
+            Element looper = element;
+
+            var absoluteX = element.X + element.Margin.Top;
+            var absoluteY = element.Y + element.Margin.Left;
+
+            // TODO: add logic to handle titles, headers, or other non-view bars
```

Comment:
Should this TODO be an issue or dropped?

Reply:
This needs a bit more work...